### PR TITLE
fix: Remove check for darwin arm64 

### LINKED ISSUE #9095: Support darwin-arm-64 in https://get.keptn.sh
When I try to install the Keptn CLI via the Quick Start instructions, I get the following message
```
curl -sL https://get.keptn.sh | bash

The newest version of Keptn is  and will be used automatically
Determined target architecture arm64
!!! Apple Silicon detected
!!! Native Apple Silicon support is currently pending, falling back to amd64 for now.
```

However, the Keptn CLI is already available also for darwin-arm64. See e.g. https://github.com/keptn/keptn/releases 

**What?** 
Quick start supports darwin-arm-64

### PR DESCRIPTION
**Pull request by RealAnna**
Signed-off-by: RealAnna <anna.reale@dynatrace.com>

closes https://github.com/keptn/keptn/issues/9095

### CODE REVIEW

File: src/get.sh [hunk 1]
```diff
@@ -67,6 +67,12 @@ runAsRoot() {
     fi
 }
 
+version_greater_equal()
+{
+    printf '%s\n%s\n' "$2" "$1" | sort --check=quiet --version-sort
```
Comment (thisthat) [inline src/get.sh]: I think we need to add the following:
```bash
return $?
```

File: src/get.sh [hunk 2]
```diff
@@ -143,7 +150,7 @@ else
     exit 1
 fi
 
-if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]]; then
+if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]] && [[ ! version_greater_equal $KEPTN_VERSION 0.9.0 ]] ; then
```
Comment (christian-kreuzberger-dtx) [inline src/get.sh]: Hi!
As I'm the one responsible for introducing the original condition here, I'd like to weigh in.
my 2 cents: This was introduced when we were lacking darwin-arm64 support (we only had darwin-amd64). Keptn [0.10.0](https://github.com/keptn/keptn/releases/tag/0.10.0) and newer releases already contain a Keptn CLI binary for darwin-arm64".
If for whatever reason the user tries to download an older release on an M1 Mac, they will run into an error in the lines below
```
curl --fail -L "${URL}" --output ${FILENAME}
curl_exit_status=$?
if [ $curl_exit_status -ne 0 ]; then
    echo "An error occured while trying to download keptn from GitHub. Please manually download a release from https://github.com/keptn/keptn/releases."
    exit $curl_exit_status
fi
```

TL;DR: In the spirit of lowering the maintenance bill, we should get rid of 
```
if [[ "$DISTR" == "darwin" ]] && [[ "$KEPTN_ARCH" == "arm64" ]]; then
    echo "!!! Apple Silicon detected"
    echo "!!! Native Apple Silicon support is currently pending, falling back to amd64 for now."
    echo "!!! You should be able to run the Keptn cli using Rosetta 2 - read https://support.apple.com/en-us/HT211861 for more information)."
    KEPTN_ARCH="amd64"
fi
```

Comment (christian-kreuzberger-dtx) [reply src/get.sh]: Add: The initial backwards-compatibility check for Keptn versions below 0.8.0 was also removed over a year ago: https://github.com/keptn/get.keptn.sh/pull/53

Comment (RealAnna) [reply src/get.sh]: Hi @christian-kreuzberger-dtx  yes I realised we do not cover any version lower then 0.12 anyway so you are right I got rid of it :)

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Review by thisthat — APPROVED** [review]
